graylog2_handler/__openerp__.py

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "GrayLog2 Handler",
+    "version": "8.0.0.0.1",
+    "author": ("Management and Accounting On-line, "
+               "Odoo Community Association (OCA)"),
+    "website": "https://github.com/OCA/server-tools/",
+    "summary": "Provides ability to send log messages to graylog2 server",
+    "category": "Added functionality",
```

Comment:
I suggest changing Category to ["Extra Tools"](https://github.com/odoo/odoo/blob/8.0/openerp/addons/base/module/module_data.xml#L83)